README.md

```diff
@@ -187,14 +187,14 @@ Is used to generate data cubes.
 
       Generate data cube. Data cubes may be created in one go or successively in
       append mode, input by input. The input may be one or more input files or a
-      pattern that may contain wildcards '?', '*', and '**'.
+      pattern that may contain wildcards '?', '*', and '**'. The input files can
+        be passed as lines of a text file.
 
     Options:
       -p, --proc INPUT_PROCESSOR      Input processor type name. The choices as
-                                      input processor are: [].  Additional
-                                      information about input processors can be
-                                      accessed by calling xcube generate_cube
-                                      --info
+                                      input processor and additional information
+                                      about input processors  can be accessed by
+                                      calling xcube gen --info .
```

Comment:
Add `defaults to "default" - the default input processor that can deal with most common datasets conforming with the CF conventions.`